tests/integration/test_cross_replication/test.py

```diff
@@ -45,7 +45,7 @@ def started_cluster():
 2017-06-16	333	2
 '''
         node1.query("INSERT INTO distributed FORMAT TSV", stdin=to_insert)
-        time.sleep(0.5)
+        time.sleep(5)
```

Comment:
???)

Reply:
I guess I had failures that data was not yet replicated before actual test.